code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -700,15 +700,14 @@ Pass a positive integer as an argument to override a bot's default speed.
 	return
 
 //
-/mob/living/simple_animal/bot/proc/bot_control_message(command,user,user_turf,user_access)

```

Comment:
Not entirely sure if there will be problems, but test this with the AI making a summon request, as it may rely on the eye feeding `user_turf`. Don't have the opportunity to check in-depth at the moment.

Reply:
The AI code path calls `call_bot` directly, `bot_control` and `bot_control_message` are only called from PDAs.